odoo/addons/base/views/res_partner_views.xml

```diff
@@ -146,13 +146,14 @@
             <field name="arch" type="xml">
                 <form string="Partners">
                 <div class="alert alert-warning oe_edit_only" role="alert" attrs="{'invisible': [('same_vat_partner_id', '=', False)]}">
-                  A partner with the same <span><span class="o_vat_label">Tax ID</span></span> already exists (<field name="same_vat_partner_id"/>), are you sure to create a new one?
+                    A partner with the same <span><span class="o_vat_label">Tax ID</span></span> already exists (<field name="same_vat_partner_id"/>). <span attrs="{'invisible': [('id', '!=', False)]}" >Are you sure to create a new one?</span>
```

Comment:
The first sentence is weird too then.
> Another partner with the same Tax ID exists

Because both already exist, and "Duh, I know that a partner with this VAT exists, I have it in front of me".

Reply:
> Because both already exist.

Not really, It is still not saved yet and can Prevent duplication.

Reply:
I was talking in the case one already exists.
You would have either
> A partner with the same Tax ID already exists (...), are you sure to create a new one?

Or
> Another partner with the same Tax ID exists (...)

Reply:
I still Believe it is okay to continue with this Change.